get_pos.py

```diff
@@ -0,0 +1,13 @@
+def get_positive_numbers() -> list:
+    """
+    Function to read inputed_numbers separated by a comma from user and returns all the positive inputed_numbers.
+    :return: positive inputed_numbers.
+    """
+    inputed_numbers = input("enter numbers separated by comma:")
```

Comment:
In this case the assignment wasn't clear but in general we prefer to seperate between IO and method logic.
https://wiki.c2.com/?SeparateIoFromCalculation